adcm_aio_client/core/host_groups/action_group.py

```diff
@@ -33,6 +34,12 @@ def actions(self: Self) -> ActionsAccessor:
 
 class ActionHostGroupNode(HostGroupNode[Union["Cluster", "Service", "Component"], ActionHostGroup]):
     class_type = ActionHostGroup
+    filtering = Filtering(
+        FilterBy("id", COMMON_OPERATIONS, int),
+        FilterByName,
+        FilterBy("owner_id", {"eq"}, int),
+        FilterBy("owner_type", {"eq"}, str),
```

Comment:
should discuss it with @a-alferov

Reply:
removed `owner_id,` `owner_type` filters from host_groups